renovate.json

```diff
@@ -9,8 +9,8 @@
   "packageRules": [
     {
       "matchUpdateTypes": ["minor"],
```

Comment:
We extend from the main `renovatebot/.github` config, which also has some rules that cover automerging packages: https://github.com/renovatebot/.github/blob/main/default.json

I'm not sure we _should_ be automerging patch updates for `mkdocs-material` though. It's hard to be sure that even a `patch` level update doesn't break any visual layout/styles.

There's a tradeoff here, if we automerge `patch` updates behind our back, we'll have less "update noise", but less certainty that things are actually okay.

What do you maintainers think?

Reply:
until now we never had issues with patch updates, so i think they are safe